lua/efmls-configs/linters/ruff.lua

```diff
@@ -0,0 +1,22 @@
+-- Metadata
+-- languages: python
+-- url: https://github.com/astral-sh/ruff
+
+local sourceText = require('efmls-configs.utils').sourceText
+local fs = require('efmls-configs.fs')
+
+local linter = 'ruff'
+local command = string.format('%s check --output-format', fs.executable(linter))
+
+return {
```

Comment:
I see that there are other severity letters in the lint message, so add a `lintSeverity = 4` in the table to catch any messages that do not match the `%t` in `lintFormats`